elasticsearch_collectd.py

```diff
@@ -563,7 +876,37 @@ def configure_callback(conf):
 def init_stats():
     global ES_HOST, ES_PORT, ES_NODE_URL, ES_CLUSTER_URL, ES_INDEX_URL, \
         ES_VERSION, VERBOSE_LOGGING, NODE_STATS_CUR, INDEX_STATS_CUR, \
-        CLUSTER_STATS_CUR, ENABLE_INDEX_STATS, ENABLE_CLUSTER_STATS
+        CLUSTER_STATS_CUR, ENABLE_INDEX_STATS, ENABLE_CLUSTER_STATS, \
+        INDEX_INTERVAL, INDEX_SKIP, COLLECTION_INTERVAL, SKIP_COUNT, \
+        DEPRECATED_NODE_STATS, THREAD_POOLS, CONFIGURED_THREAD_POOLS
+
+    # Sanitize the COLLECTION_INTERVAL and INDEX_INTERVAL
```

Comment:
let's move this into a function

INDEX_INTERVAL  = round_index_interval(collection_interval, configured_index_interval)